packages/scene-composer/tests/components/toolbars/floatingToolbar/FloatingToolbar.spec.tsx

```diff
@@ -44,4 +56,36 @@ describe('FloatingToolbar', () => {
     const container = create(<FloatingToolbar enableDefaultItems={true} />);
     expect(container).toMatchSnapshot();
   });
+
+  it('should display the toolbar vertically if the canvas is big', async () => {
+    //set up very small canvas so the toolbar flips to horizontal
```

Comment:
Incorrect comment. Please remove this.
Also, comment not needed in the next test case as we are indicating in the test itself that canvas is small.

Reply:
Fixed, I missed this while reading through changes.